docs/packages/omc3/workflow.md

```diff
@@ -1,7 +1,841 @@
+    More than one element can be given to this flag.
+
+The converter will create new files with the `.sdds` suffix appended to the original filename.
+In our case, a `trackone.sdds` file will be created.
+
+!!! tip "Other Uses"
+    The `tbt_converter` provides additional functionality such as adding noise to the provided data or averaging across particles.
+    To use these options, refer to the [converter's API documentation][tbt_converter].
+
+## Creating a Model
+
+In order to perform the optics analysis, one needs a model of the given machine to compare to.
+For this, `omc3` provides the `model_creator` entrypoint, which allows you to run a model simulation of the desired machine and output the needed files.
+
+??? example "Supported Machines"
+    The out-of-the-box supported machines for model creation are `lhc`, `ps` and `psbooster`, machines we work on.
+    While the `skekb`, `JPARC`, `petra` and `iota` have accelerator classes, no model creator has been implemented for them yet.
+    It is possible to extend this list for your machine by defining an appropriate `Accelerator` class as well as a model creator.
+    See [this guide][new_machine_guide] for implementation steps.
+
+In our example, we would like to compare our data to the nominal model of the 2018 LHC. 
+Using the script to create a nominal model of the 2018 LHCB1, with the machine configuration and opticsfile used in our example, goes as:
+```bash
+python -m omc3.model_creator \
+    --accel lhc \
+    --type nominal \
+    --year 2018 \
+    --beam 1 \
+    --energy 6.5 \
+    --nat_tunes 62.31 60.32 \
+    --modifiers /afs/cern.ch/eng/lhc/optics/runII/2018/PROTON/opticsfile.22 \
+    --outputdir lhc_model
+```
+
+!!! bug "Specifying Paths"
+    At the moment, some issues may arise down the line when specifying input files with relative paths.
+    To ensure a seamless workflow, it is recommended to use absolute paths.
+    This will be dealt with in the next release.
+
+Some of these options belong to the `model_creator` itself, while others depend explicitely on the chosen machine, here the LHC.
+Refer to the [model creator's API documentation][model_creator] for the list of options.
+
+The model creation runs a `MAD-X` scenario and outputs the relevant twiss results to the desired directory.
+Running `ls lhc_model/` yields:
+```bash
+b2_errors.tfs		error_deffs.txt		    macros			twiss_elements.dat
+b2_settings.madx	job.create_model.madx	twiss.dat
+```
+
+!!! question "What is a Model?"
+    As one can see, a "model" is essentially one or more TFS files with optics functions at BPMs (`twiss.dat`) and elements (`twiss_elements.dat`), other files being here for the user to understand or reproduce the result.
+    Had we created a driven model, then an additional `twiss_ac.dat` or `twiss_adt.dat` file would have been created, with optics functions at BPMs while driving the beam.
```

Comment:
maybe add a comment on what a driven model is .. that is, basically a model in which the AC-Dipole effect on the optics has been taken into account, which is not default in mad-x and has to be done via some tricks, see https://github.com/pylhc/Beta-Beat.src/blob/master/madx/lib/lhc.macros.madx

Reply:
Great suggestion!